packages/foundation/src/hooks/use-viewport.ts

```diff
@@ -0,0 +1,25 @@
+import { useWindowDimensions } from './use-window-dimensions';
+import { BreakpointsSizes } from '../responsive/breakpoints-sizes';
+
+type useViewportResponse = {
+  isSmall: boolean;
+  isMedium: boolean;
+  isLarge: boolean;
+  isXLarge: boolean;
+};
+
+export const useViewport = (): useViewportResponse => {
```

Comment:
I'm creating this hook so we can access to the breakpoint identifier from js rather than just jsx

Reply:
Would you have an example of an use case like this?